apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.sql

```diff
@@ -50,9 +50,10 @@ FROM
   LEFT JOIN assessment_instances AS ai ON (ai.id = iq.assessment_instance_id)
   LEFT JOIN assessments AS a ON (a.id = ai.assessment_id)
   LEFT JOIN pl_courses AS c ON (c.id = v.course_id)
-  LEFT JOIN course_instances AS ci ON (ci.course_id = c.id)
+  LEFT JOIN course_instances AS ci ON (ci.id = v.course_instance_id)
   LEFT JOIN users AS u ON (u.user_id = s.auth_user_id)
 WHERE
   gj.id = $job_id
   AND gj.grading_method = 'External'
-  AND ci.id = $course_instance_id;
+  AND c.id = $course_id
+  AND ci.id IS NOT DISTINCT FROM $course_instance_id;
```

Comment:
We want our files to have trailing newlines. If you haven't already, I'd recommend getting the EditorConfig extension for VS Code: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig. This should ensure that your editor adds them automatically.